src/mstatements.jl

```diff
@@ -18,13 +18,7 @@ function write_mstatement(io::IO, s::Char)
     print(io, "'")
 end
 
-
-# Hack to support Julia 0.2 and 0.3
-const matlab_infix_ops = union!(Set{Symbol}(),
-    [:+, :-, :*, :/, :\, :^,
-     :.+, :.-, :.*, :./, :.\, :.^,
-     :|, :&])
-
+const matlab_infix_ops = Set{Symbol}([:+, :-, :*, :/, :\, :^, :.+, :.-, :.*, :./, :.\, :.^, :|, :&])
```

Comment:
This can just be `Set([:+, :-, :*, :/, :\, :^, :.+, :.-, :.*, :./, :.\, :.^, :|, :&])` since Julia can figure out it should be a `Set{Symbol}`.